common/src/Microsoft.Azure.IIoT.Hub.Module.Client/src/Default/IoTSdkFactory.cs

```diff
@@ -207,469 +226,32 @@ public sealed class IoTSdkFactory : IClientFactory, IDisposable {
         /// <returns></returns>
         private Task<IClient> CreateAdapterAsync(string product, Action onError,
             ITransportSettings transportSetting = null) {
-            if (string.IsNullOrEmpty(ModuleId)) {
-                if (_cs == null) {
-                    throw new InvalidConfigurationException(
-                        "No connection string for device client specified.");
-                }
-                return DeviceClientAdapter.CreateAsync(product, _cs, DeviceId,
-                    transportSetting, _timeout, RetryPolicy, onError, _logger);
-            }
-            return ModuleClientAdapter.CreateAsync(product, _cs, DeviceId, ModuleId,
-                transportSetting, _timeout, RetryPolicy, onError, _logger);
-        }
 
-        /// <summary>
-        /// Adapts module client to interface
-        /// </summary>
-        public sealed class ModuleClientAdapter : IClient {
-
-            /// <summary>
-            /// Whether the client is closed
-            /// </summary>
-            public bool IsClosed { get; internal set; }
-
-            /// <summary>
-            /// Create client
-            /// </summary>
-            /// <param name="client"></param>
-            internal ModuleClientAdapter(ModuleClient client) {
-                _client = client ??
-                    throw new ArgumentNullException(nameof(client));
-            }
+            var timeout = TimeSpan.FromSeconds(15);
```

Comment:
Please make this timeout configurable, default 5min (like before) when used with MQTT Broker (and w/o IoT Hub connector) this value have to be reduced to be operational